pkg/httpserver/httpserver_test.go

```diff
@@ -59,17 +60,21 @@ var _ = Describe("Test httpserver", func() {
 			}),
 		})
 
-		httpServ := NewHTTPServer(testProbes, nil, metricsStore, testPort, mockDebugServer)
+		httpServ := NewHTTPServer(testProbes, nil, metricsStore, testPort)
```

Comment:
let's write the test for the new debug server function into another test function